travis/templates/mysql/dev-life-export.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Since I'm nitpicking about the JSON indentation, don't forget to change the indentation here too

Reply:
That's the same indentation, I must change both or the build will fail.